prusti-interface/src/prusti_error.rs

```diff
@@ -36,6 +37,7 @@ impl PrustiError {
             span,
             help: None,
             note: None,
+            additional_notes: vec![]
```

Comment:
Please be consistent about the trailing commas, it's easier to edit code if there is always a trailing comma in a multiline call or declaration.